### DISCUSSION TITLE
I haven't been able to play Osu!lazer for almost a year now.

### DISCUSSION THREAD

**Discussion by LoremIvashkov** [ppy/osu#17733] [Q&A]
Whenever I try to load up Osu!lazer, it doesn't even open. I've tried redownloading Osu! aswell as Osu!lazer. All attempts to fix it have been an absloute failure. If anyone can help me, it'd be a huge help. I'll attach my SquirrelSetup.Log file below:
[SquirrelSetup.log](https://github.com/ppy/osu/files/8455715/SquirrelSetup.log)

**Comment by peppy** [comment]
For some reason the installer doesn't have permission to delete the files of the previous install. You may need to do this manually in `%appdatalocal%\osulazer`. Deleting all files should be fine as your data is stored elsewhere.
